wordcloud/wordcloud.py

```diff
@@ -503,16 +508,17 @@ def process_text(self, text):
 
         flags = (re.UNICODE if sys.version < '3' and type(text) is unicode
                  else 0)
-        regexp = self.regexp if self.regexp is not None else r"\w[\w']+"
+        regexp = self.regexp if self.regexp is not None else r"(\w[\w']+|\w+)"
```

Comment:
Can you explain your change? I think I wouldn't change the current default behavior. Is the intent that you want single digits to show? We could change it when  render_numbers is false but that would create dependencies between the parameters, which I don't like.

Maybe the simplest solution is to undo this change and just put this as a custom regexp in the test? Or use longer numbers in the test?
Thanks!